src/mjolnir/graphtilebuilder.cc

```diff
@@ -67,6 +67,7 @@ GraphTileBuilder::GraphTileBuilder(const std::string& tile_dir,
 
     // Add a dummy admin record at index 0 to be used if admin records are
     // not used/created or if none is found.
+    // TODO: do we really want to hardcode "None" for country and state?
```

Comment:
@kevinkreiser I would remove this todo.  Yes we always want something at 0 and all real admins start at 1.  0 is used if no admin is found or the admin db is not used.

Reply:
oh.  I think at the time we just agreed to make it None as I think we were seeing blank admin names.